components/buttons.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from "react";
+
+interface Props {
+    text: string
```

Comment:
Is it 2 spaces as indentation ?

Reply:
No, it doesn't have any space in those lines

Reply:
done